tests/testthat/test-MltplxExperiment.R

```diff
@@ -226,13 +293,24 @@ test_that("Window size parameter works", {
   window_sizes3 <- mutate(window_sizes,
                          max_x = max_x - 25)
 
-  expect_error({
-    mltplx_exp <- new_MltplxExperiment(x = cell_x_values,
-                                       y = cell_y_values,
-                                       window_sizes = window_sizes3,
-                                       marks = factor(cell_marks),
-                                       slide_id = slide_ids)
-  })
+  # For some reason this test started breaking, because purrr
```

Comment:
can we revisit this test sometime?

Reply:
I actually fixed this in the other branch I believe; I assume it will generate a merge conflict when one is merged, which I'll then resolve